src/System.IdentityModel.Tokens.Jwt/JwtSecurityTokenHandler.cs

```diff
@@ -382,7 +383,7 @@ public virtual string CreateEncodedJwt(string issuer, string audience, ClaimsIde
         /// <exception cref="ArgumentException">If 'expires' &lt;= 'notBefore'.</exception>
         public virtual string CreateEncodedJwt(string issuer, string audience, ClaimsIdentity subject, DateTime? notBefore, DateTime? expires, DateTime? issuedAt, SigningCredentials signingCredentials, EncryptingCredentials encryptingCredentials)
```

Comment:
Do we need to add a input parameter in CreateEncodedJwt to provide an option to pass in SecurityTokenDescriptor.Claims?

Reply:
I think it might be beneficial to add overloaded methods for both CreateEncodedJwt() and CreateJwtSecurityToken() that take in a Dictionary<string, object> instead of a ClaimsIdentity.

Reply:
Good idea. Don't we need both ClaimsIdentity and Dictionary<string, object>?